ui/js/dfv/src/config/prop-types.js

```diff
@@ -342,13 +353,25 @@ export const FIELD_PROP_TYPE_SHAPE = PropTypes.exact( {
 		PropTypes.arrayOf( PropTypes.string ),
 	] ),
 	pick_select_text: PropTypes.string,
-	pick_show_edit_link: PropTypes.string,
-	pick_show_icon: PropTypes.string,
-	pick_show_view_link: PropTypes.string,
+	pick_show_edit_link: PropTypes.oneOfType( [
+		PropTypes.number, // Could be 0 if not set.
+		PropTypes.string,
+	] ),
+	pick_show_icon: PropTypes.oneOfType( [
```

Comment:
Should this be NUMBER_OR_NUMBER_AS_STRING?

Reply:
Yeah, that's probably better here, although a consistent type from the back-end would be nice